### DISCUSSION TITLE
License for ATN trajectory data

### DISCUSSION THREAD

**Discussion by MathewBiddle** [ioos/ioos-atn-data#2] [Q&A]
OBIS requires CC-0, CC-BY, or CC-BY-NC and NCEI might be alright with a use statement similar to what has been used for the IOOS Glider DAC archive.
> license: The data may be used and redistributed for free but is not intended for legal use, since it may contain inaccuracies. No person or group associated with this data makes any warranty, express or implied, including warranties of merchantability and fitness for a particular purpose, or assumes any legal liability for the accuracy, completeness, or usefulness, of this information.

The question is, how do we proceed? What, if any, statement should the ATN DAC put in the license attribute of the netCDF files?

**Comment by AbbyBenson** [comment]
Quick clarification that OBIS requires [CC-0](https://creativecommons.org/share-your-work/public-domain/cc0), [CC-BY](https://creativecommons.org/licenses/by/4.0/), or [CC-BY-NC](https://creativecommons.org/licenses/by-nc/4.0/). More information on the OBIS data policy [here](https://obis.org/manual/policy/).

**Reply by MathewBiddle** [reply]
thank you for that clarification! I have updated the discussion to reflect this detail.

**Comment by MathewBiddle** [comment]
Out of curiosity what was the approach taken with the AMBON observations [archived at NCEI](https://www.ncei.noaa.gov/access/metadata/landing-page/bin/iso?id=gov.noaa.nodc:0208100) and available through [OBIS](https://obis.org/dataset/2d60dbdc-e801-4c66-991e-d484e8b08227)? At what point was the CC applied?

Also, in reviewing the [OBIS data policy](https://obis.org/manual/policy/) I read the following statement as not a requirement from the data provider to apply a license. I read this as an acknowledgment from the provider that OBIS will apply one of those licenses to the data. Is `it` referring to OBIS node or data provider?

>The OBIS nodes warrant that they have made the necessary agreements with the original data providers that it can make the data available to OBIS data under the following Creative Commons licenses:
> * CC-0
> * CC-BY
> * CC-BY-NC
>
> CC-0 is the preferred one and CC-BY-NC the least preferred.

**Reply by AbbyBenson** [reply]
I have to admit that I'm not entirely sure what it is you're trying to determine and some of the language in your question seems like splitting hairs? We might need to bring Axiom (Adrienne Canino) here to fully answer your first question but as an OBIS node manager I do not apply data licenses to data without consent from the person supplying me the data to pick one of those licenses. The IPT is setup such that I cannot publish data without selecting one of those three licenses. When I am working with someone to publish their data to OBIS one of the questions I tell them is a requirement is to select one of those three licenses. If they do not select one then I can't publish the data. Attached is a screenshot of what it looks like in the IPT. It is a required field (note the *) and the software will not let me select "No license selected". It was a requirement that was [adopted by GBIF](https://www.gbif.org/news/82812/licensing-milestone-for-data-access-in-gbiforg) and OBIS back in 2016. Because I'm unsure what you're trying to determine, I'm not sure if this answers your question.
<img width="383" alt="IPT_DataLicense" src="https://user-images.githubusercontent.com/7586150/114413035-de19bf00-9b6a-11eb-9710-ff6529dd27b4.PNG">

**Reply by MathewBiddle** [reply]
This helps! I'm trying to determine a way for the requirements for all three parties can be met. I think there is some confusion as to when the license needs to be applied for OBIS. The distinction here is that OBIS requires **consent** to apply one of the three licenses. OBIS does not require those licenses to be applied by the data provider before data gets to OBIS. 

> as an OBIS node manager I do not apply data licenses to data without consent from the person supplying me the data to pick one of those licenses. 

I'm wondering if it would be sufficient to have ATN consent to one of those 3 options for OBIS data publishing, when we develop the transfer mechanism to submit data to OBIS? That consent wouldn't need to be in the data files themselves, but maybe some sort of submission agreement between OBIS and ATN?

Would it be sufficient for ATN to say "Yes, you can select CC0 for the datasets submitted to OBIS". Then, ATN can insert this statement (or something similar) in the netCDF files to be submitted to NCEI?
>The data may be used and redistributed for free but is not intended for legal use, since it may contain inaccuracies. No person or group associated with this data makes any warranty, express or implied, including warranties of merchantability and fitness for a particular purpose, or assumes any legal liability for the accuracy, completeness, or usefulness, of this information.

**Reply by MathewBiddle** [reply]
@mmckinzie If you and Bill can decide which license OBIS should apply when they **serve** the data, I think we are good here.  

Think of it as if you were submitting one of these trajectory files to OBIS. Which license would you have Abby pick? [CC-0](https://creativecommons.org/share-your-work/public-domain/cc0), [CC-BY](https://creativecommons.org/licenses/by/4.0/), or [CC-BY-NC](https://creativecommons.org/licenses/by-nc/4.0/)?

As for NCEI, including a statement similar to what is above is sufficient. May need to have NCEI review it first, but that type of statement is okay.

So, I think once that text is solidified we can move forward.

**Reply by mmckinzie** [reply]
Hey Matt and Abby. Sorry for the delay in response. I am working on getting answers to above questions from Bill and the ATN SG. As of now it appears the SG does still prefer OBIS apply a CC-BY license over CC-0 but again still waiting on final consensus. I have drafted an ATN statement for NCEI archive and waiting on feedback from Bill and the SG. Once finalized, hopefully, by end of this week, I will share our generic statement with the group.

**Reply by MathewBiddle** [reply]
Current ATN license statement:
>These data may be used and redistributed for free, but are not intended for legal use, since they may contain inaccuracies. No person or group associated with these data makes any warranty, expressed or implied, including warranties of merchantability and fitness for a particular purpose, or assumes any legal liability for the accuracy, completeness or usefulness of this information. This disclaimer applies to both individual use of these data and aggregate use with other data. It is strongly recommended that users read and fully comprehend associated metadata prior to use. Please acknowledge the U.S. Animal Telemetry Network (ATN) or the specified citation as the source from which these data were obtained in any publications and/or representations of these data. Communication and collaboration with dataset authors are strongly encouraged.
